src/domain/orders/details/utils/create-filtering.js

```diff
@@ -0,0 +1,54 @@
+export const filterItems = (order, is_claim) => {
+  let tmpp = order.items.reduce(
+    (map, obj) => map.set(obj.id, { ...obj }),
+    new Map()
+  )
+
+  let claim_items = []
+
+  if (order.claims && order.claims.length) {
+    for (const s of order.claims) {
```

Comment:
Please camelCase all variables in this file :)

Reply:
you got it :)